tmux/multimodal_activity_recognition_debug.yml

```diff
@@ -0,0 +1,98 @@
+#
+# Fragment of the system that includes data capture, object detection, and
+# debug visualization of the detections in the image space.
+#
+
+name: 20220719-darpa-sitevisit-coffee-demo
```

Comment:
This config and the `activity_detection_debug.yml` probably serve the same purpose. Let's consolidate them instead of having two different configs that do basically the same thing.

Reply:
One is for the learn activity recognition and the other is for sri's multimodel model. Is it useful to have both of them?

Reply:
I don't think it is, because the difference is only a single node that we can swap (we can leave the unused on commented out for now). In an ideal future, we have an index in the documentation of all the nodes that we have to inform someone on what can be added/removed here.